### PR TITLE
[Win] Fix compilation after fullscreen API

### PR DESCRIPTION
**Pull request by vitallium**

### CODE REVIEW

File: Source/WebKit/qt/Api/qwebfullscreenrequest.cpp [hunk 1]
```diff
@@ -28,10 +28,9 @@
 
 #include "QWebPageAdapter.h"
 
-#include <QPointer>
-#include <WebCore/Element.h>
+#include <Element.h>
```
Comment (annulen) [inline Source/WebKit/qt/Api/qwebfullscreenrequest.cpp]: Either `#include "Element.h"` or `#include <WebCore/Element.h>`, note that WebKit2 code and port-independent WebKit code uses the latter for WebCore headers

File: Source/WebKit/qt/Api/qwebfullscreenrequest.h [hunk 2]
```diff
@@ -28,8 +28,13 @@
 
 #include <QtCore/qscopedpointer.h>
 #include <QtCore/qurl.h>
-#include <QtWebKit/qwebelement.h>
-#include <QtWebKit/qwebkitglobal.h>
+
+#include "qwebelement.h"
```
Comment (annulen) [inline Source/WebKit/qt/Api/qwebfullscreenrequest.h]: Public API headers refer to other API headers with module name, e.g. `#include <QtWebKit/qwebelement.h>`. Look at any Qt public header.

Comment (Vitallium) [inline Source/WebKit/qt/Api/qwebfullscreenrequest.h]: Doing like this `#include <QtWebKit/qwebelement.h>` breaks the Win build.

Comment (Vitallium) [inline Source/WebKit/qt/Api/qwebfullscreenrequest.h]: Ideally this include should not be here. But `Q_DECLARE_TYPEINFO` requires the full type info.

File: Source/WebKit/qt/Api/qwebfullscreenrequest.cpp (hunk 1 above)
Comment (Vitallium) [inline Source/WebKit/qt/Api/qwebfullscreenrequest.cpp]: I'll use the first to match all Qt files in `api`

File: Source/WebKit/qt/Api/qwebfullscreenrequest.h (hunk 2 above)
Comment (annulen) [inline Source/WebKit/qt/Api/qwebfullscreenrequest.h]: In Source/WebKit/CMakeLists.txt "${DERIVED_SOURCES_DIR}/ForwardingHeaders" is addeded to WebKit_INCLUDE_DIRECTORIES, so QtWebKit and QtWebKitWidgets headers there should be visible to compiler.

qwebelement.h is indeed no needed, but qwebkitglobal.h still needs to be here

**Comment by Vitallium** [comment]
Now it builds fine.

**Pull request merged**